packages/api-explorer/CHANGELOG.md

```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [21.1.0](https://www.github.com/looker-open-source/sdk-codegen/compare/api-explorer-v21.0.10...api-explorer-v21.1.0) (2021-03-25)
+
+
+### Features
+
+* Added --versions option to the code generator ([#514](https://www.github.com/looker-open-source/sdk-codegen/issues/514)) ([ee6f3e8](https://www.github.com/looker-open-source/sdk-codegen/commit/ee6f3e8f55e300df1a75c9be89b47f067bc08dee))
+
+
+### Dependencies
+
+* The following workspace dependencies were updated
+  * dependencies
+    * @looker/run-it bumped from ^21.0.10 to ^21.1.0
+    * @looker/sdk bumped from ^21.0.10 to ^21.4.0
+    * @looker/sdk-codegen bumped from ^21.0.10 to ^21.0.11
+  * devDependencies
+    * @looker/sdk-codegen-scripts bumped from ^21.0.9 to ^21.0.11
+    * @looker/sdk-node bumped from ^21.0.10 to ^21.4.0
+
 ## [Current]
```

Comment:
I'll update this to `### [21.0.10]` in a separate PR and we'll re-run release-please afterwards

Reply:
https://github.com/looker-open-source/sdk-codegen/pull/536